### ISSUE TITLE
linode-cli fails to create a linode with a label

### ISSUE THREAD

**Issue by patthiel** [linode/linode-cli#31]
Attempting to create a linode per the example in the [readme](https://github.com/linode/linode-cli/blob/master/README.rst) fails.

```bash
linode-cli linodes create --type g5-standard-2 --region us-east --image linode/debian9 --label cli-1 --root_pass hunter7

linode-cli: error: unrecognized arguments: --label cli-1
```
It seems support for the `--label` argument was never added

**Comment by g1bl3t** [comment]
Having the same problem after upgrading to the latest version

**Comment by g1bl3t** [comment]
--label was working for me on 2.0.0, but now having the same issue after upgrading to 2.0.4

**Comment by Dorthu** [comment]
Looks like these are missing from the spec, too: https://developers.linode.com/api/v4#operation/createLinodeInstance.  Thanks for the report, I'm adding these in everywhere and rebuilding the CLI.

**Comment by Dorthu** [comment]
This is fixed in version `2.0.5` - a `pip install --upgrade linode-cli` should have you up and running.  Sorry for the inconvenience.
